Doc/README.txt

```diff
@@ -104,6 +104,14 @@ Then, from the ``Doc`` directory, run ::
 where ``<builder>`` is one of html, text, latex, or htmlhelp (for explanations
 see the make targets above).
 
+Deprecation header
+==================
+
+You can define the ``outdated`` variable in ``html_context`` to show a
```

Comment:
Since we're merging this change into 2.7 at the time it is being end-of-lifed, the "outdated" variable should be always set when building the 2.7 docs and not depend on being passed in from the "make" command.  (We also build the docs in various ways so it would be problematic to modify all the make calls.)  I'm not sure where the best place to put the definition is; just so it ends up in the branch.